mozilla-release/browser/components/AutoPrivateTab.js

```diff
@@ -365,7 +365,7 @@ AutoPrivateTabDatabase.prototype = {
               "apt.cleanupPrompt.learnMoreButton"),
           callback: (notification, descr) => {
             gBrowser.ownerGlobal.openLinkIn(
-                "https://cliqz.com/support/daten-vergessen-modus",
+                "https://cliqz.com/support/automatic-forget-mode",
```

Comment:
@maxim-cliqz We have both German and English for this link in other places, but we only use German link here